gbpservice/nfp/configurator/drivers/firewall/paloalto/paloAlto_fw_driver.py

```diff
@@ -0,0 +1,875 @@
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
```

Comment:
why not paloalto_fw_driver.py or palto_fw_driver.py as file name, just to follow the same convention of naming?